src/actions.ml

```diff
@@ -791,20 +1295,26 @@ let pipeline_action ~bot_info pipeline_info ~gitlab_mapping : unit Lwt.t =
                 ?conclusion ~title ~details_url:pipeline_url ~summary
                 ~external_id ()
               >>= fun () ->
-              Lwt_unix.sleep 5.
-              >>= fun () ->
-              match
-                ( owner
-                , repo
-                , pipeline_info.state
-                , pipeline_info.common_info.base_commit )
-              with
-              | "coq", "coq", "failed", Some base_commit ->
-                  minimize_failed_tests ~bot_info ~owner ~repo ~base:base_commit
-                    ~head:pipeline_info.common_info.head_commit ~pr_number
-                    ~head_pipeline_summary:summary
-              | _ ->
-                  Lwt.return () ) ) )
+              if false then
+                (* We don't want to do this until the criterion for
+                   minimization stabilizes, but we want OCaml to keep
+                   checking that this code typechecks *)
```

Comment:
What do you mean? We want to test the minimization suggestion feature, don't we?

Reply:
I've added support for having the bot run jobs automatically, in addition to suggesting that jobs be run. Currently the conditions under which the bot runs tests automatically is `false`, and OCaml was complaining that my constructor was unused.

Reply:
OK but my problem is that with this `if false`, you do not call the minimization code anymore (be it in auto or in suggestion mode).

Reply:
Ah, sorry, I was confused about where this comment was and had forgotten that I left this `if false` in.  Good catch, I'll remove it.

Reply:
Updated